### PR TITLE
Update tag partial

### PR DESCRIPTION
**Pull request by JMurphyWeb**
#662 

Just change the look of the tag partial

### CODE REVIEW

File: public/css/main.css [hunk 1]
```diff
@@ -539,17 +539,17 @@ Light-teal:     #66d9e0;
...
-    background-color: #66d9e0; /* Light-teal */
-    border: 1px solid #ffffff;
-    border-radius: 3rem;
-    color: #ffffff;
-    cursor: pointer;
-    text-align: center;
-    text-transform: capitalize;
+  display: inline-block;
+  margin: 0 0.125rem 0.425rem 0;
+  padding: 0.5625rem 0.8rem;
```
Comment (harrygfox) [inline public/css/main.css]: my preference for maximum readability is for values like these to be converted in an inline comment
```css
padding: 0.5625rem 0.8rem; /* 9px 12.8px */
```
Can also serve as a nice check too: _do I definitely want a fractional pixel here? (only screens with higher dpi will get to appreciate that subtlety)_

Comment (JMurphyWeb) [inline public/css/main.css]: Do we not want to cater for those screens?

File: public/css/main.css [hunk 2]
```diff
@@ -539,17 +539,17 @@ Light-teal:     #66d9e0;
...
-    color: #ffffff;
-    cursor: pointer;
-    text-align: center;
-    text-transform: capitalize;
+  display: inline-block;
+  margin: 0 0.125rem 0.425rem 0;
+  padding: 0.5625rem 0.8rem; /* 9px 12.8px */
+  line-height: 1rem;
+  background-color: #66d9e0;
+  border: 1px solid #ffffff;
```
Comment (harrygfox) [inline public/css/main.css]: Why does this exist?

File: public/css/main.css [hunk 3]
```diff
@@ -539,17 +539,17 @@ Light-teal:     #66d9e0;
...
+  border-radius: 0.6rem;
```
Comment (harrygfox) [inline public/css/main.css]: The wireframes ask for 6px of radius which, **with a 1px border**, should be upped to 7px i.e. 0.4375rem. If you don't need the border then 0.375rem

File: public/css/main.css (hunk 2 above)
Comment (JMurphyWeb) [inline public/css/main.css]: not sure?

File: public/css/main.css (hunk 3 above)
Comment (JMurphyWeb) [inline public/css/main.css]: ok i removed the border and fixed the br.

Thanks for the precise review!

**Comment by JMurphyWeb** [comment]
Thanks @harrygfox 🌮

**Pull request merged**